### PR TITLE
Added pipeline to save held out chromosomes and cell types

### PR DESCRIPTION
**Pull request by akmorrow13**

### CODE REVIEW

File: src/main/scala/net/akmorrow13/endive/EndiveConf.scala [hunk 1]
```diff
@@ -91,6 +91,11 @@ class EndiveConf extends Serializable {
 
   @BeanProperty var sample: Boolean = true
 
+  /* Configuration values used for saving test set */
+  @BeanProperty var tf: String = null
```
Comment (Vaishaal) [inline src/main/scala/net/akmorrow13/endive/EndiveConf.scala]: Change this to test_tf or something?

File: src/main/scala/net/akmorrow13/endive/EndiveConf.scala [hunk 2]
```diff
@@ -91,6 +91,11 @@ class EndiveConf extends Serializable {
...
+  @BeanProperty var cellTypes: String = null
```
Comment (Vaishaal) [inline src/main/scala/net/akmorrow13/endive/EndiveConf.scala]: test_cellTypes here?

File: src/main/scala/net/akmorrow13/endive/EndiveConf.scala [hunk 3]
```diff
@@ -91,6 +91,11 @@ class EndiveConf extends Serializable {
...
+  @BeanProperty var hasSequences: Boolean = true
```
Comment (Vaishaal) [inline src/main/scala/net/akmorrow13/endive/EndiveConf.scala]: When do we not have sequences?

Comment (akmorrow13) [inline src/main/scala/net/akmorrow13/endive/EndiveConf.scala]: This is for dataset creation pipeline. If sequences are saved that step is skipped

File: src/main/scala/net/akmorrow13/endive/EndiveConf.scala (hunk 1 above)
Comment (akmorrow13) [inline src/main/scala/net/akmorrow13/endive/EndiveConf.scala]: no these are for saving specific tfs in dataset creation pipeline

**Pull request merged**